src/test/java/step1/SetTest.java

```diff
@@ -28,23 +28,20 @@ void setUp() {
     @DisplayName("Set의 size() 메소드를 활용해 Set의 크기를 확인")
     @Test
     void size() {
-        // given, when, then
         assertThat(numbers).hasSize(3);
     }
 
     @DisplayName("Set의 contains() 메소드를 활용해 1, 2, 3의 값이 존재하는지를 확인")
     @ParameterizedTest
     @ValueSource(ints = {1, 2, 3})
     void contains(int number) {
-        // given, when, then
-        assertThat(numbers.contains(number)).isTrue();
+        assertThat(numbers.contains(number));
```

Comment:
이 형태로는 테스트 코드의 의미가 없을 것 같습니다. 실제 포비님이 리뷰를 주신 부분도, 
```suggestion
        assertThat(numbers).contains(number);
```
와 같이 isTrue를 사용하지 않고 바로 contains로 참/거짓을 판단하는 형태입니다..!